src/css/index.css

```diff
@@ -1,15 +1,181 @@
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i,900,900i');
+@import url('https://fonts.googleapis.com/css?family=Source+Serif+Pro:300,300i,400,400i,600,600i,700,700i,900,900i');
+
 /**
  * This injects Tailwind's base styles and any base styles registered by
  * plugins.
  */
 @tailwind base;
 
+@font-face {
+    font-family: Stanford;
+    src:url("https://www-media.stanford.edu/assets/fonts/stanford.woff") format("woff"),
+    url("https://www-media.stanford.edu/assets/fonts/stanford.ttf") format("truetype");
+    font-weight:300;
+}
+
 /**
  * This injects Tailwind's component classes and any component classes
  * registered by plugins.
  */
 @tailwind components;
 
+@screen md {
+    body {
+        font-size: 1.8rem;
+    }
+}
+
+@screen 2xl {
+    body {
+        font-size: 1.9rem;
+    }
+}
+
+.su-logo {
+    @apply
+    inline-block
+    font-regular font-stanford
+    text-white;
+
+    line-height: 0.75;
+    -ms-font-feature-settings: "liga" 1;
+    font-feature-settings: "liga";
+    font-variant-ligatures: discretionary-ligatures;
+}
```

Comment:
Can these be moved into a component plugin?

Ideally, we would want everything to be in a plugin for maximum re-use and extensibility.

Reply:
Agree - will do :)